src/media/headset.ts

```diff
@@ -54,7 +54,7 @@ export class SdkHeadset {
      * @returns Promise<void>
      */
     retryConnection (micLabel: string): Promise<void> {
-        return this.headsetLibrary.retryConnection(micLabel);
+        return micLabel && this.headsetLibrary.retryConnection(micLabel.toLowerCase());
```

Comment:
If `micLabel` is falsy, you will be returning `false` and _NOT_ a Promise. Change this function to `async` if you always want it to return a promise. 

Also, it feels like a little bit of a code smell that callers need to `label.toLowerCase()` themselves? It seems like if the headset lib wants/expects lower case values, it should `toLowerCase()` those itself.